crates/starknet-server/src/api/json_rpc/spec_reader/spec_schemas/mod.rs

```diff
@@ -0,0 +1,55 @@
+use serde::{Deserialize, Serialize};
+
+use self::all_of_schema::AllOf;
+use self::array_primitive::ArrayPrimitive;
+use self::boolean_primitive::BooleanPrimitive;
+use self::integer_primitive::IntegerPrimitive;
+use self::object_primitive::ObjectPrimitive;
+use self::one_of_schema::OneOf;
+use self::ref_schema::Reference;
+use self::string_primitive::StringPrimitive;
+
+pub(crate) mod all_of_schema;
+pub(crate) mod array_primitive;
+pub(crate) mod boolean_primitive;
+pub(crate) mod integer_primitive;
+pub(crate) mod object_primitive;
+pub(crate) mod one_of_schema;
+pub(crate) mod ref_schema;
+pub(crate) mod string_primitive;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+pub struct Common {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub title: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub description: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(rename = "additionalProperties")]
```

Comment:
Is the rename necessary if already using `rename_all` above?

Reply:
[006a70d](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/230/commits/006a70d6e4faf99b0a326e687a67029bb93fbba9)